pages/about.js

```diff
@@ -120,6 +120,16 @@ export default () => (
             members of the military community.
           </p>
         </ImageCard>,
+        <ImageCard
+          alt="Operation Code Podcast"
+          className={styles.ImageCard}
+          imageSource={`${s3}photo-podcast-1.jpg`}
+        >
+          <a href="/podcasts">
```

Comment:
Check out other usages of next's `Link` component throughout the app. You'll want to change this instance!

Reply:
Instance updated in 496e113